string droneSn
DroneWayPoint[] way_point